### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by kdheepak**
This PR auto documents all the modules and makes some changes to document the new API.

### CODE REVIEW

File: docs/src/flags.md [hunk 1]
```diff
@@ -10,20 +10,20 @@ Several flag variables are also defined, each with multiple entries defined.
 Here is an example of use of `CapControlModes.KVAR`:
 
 ```julia
-CapControls.Mode(CapControlModes.KVAR)
+CapControls.Mode(Lib.CapControlModes_KVAR)
```
Comment (tshort) [inline docs/src/flags.md]: Also need `CapControlModes_KVAR` in the line above.

File: docs/src/lowlevel.md [hunk 2]
```diff
@@ -1,32 +1,9 @@
...
-end
-println(DSSCore.DSSLoads(4, 0))
-```
-
-Integer (I), floating-point (F), string (S), and variant (V) functions are
-supported. The low-level API functions are not exported.
-
-This low-level API also includes several functions for directly accessing and
-controlling an OpenDSS solution. These functions include:
+This are some of the low level functions implemented in the `OpenDSSDirect.YMatrix` module.
```
Comment (tshort) [inline docs/src/lowlevel.md]: "This are" -> "These are"

Comment (kdheepak) [reply docs/src/lowlevel.md]: Resolved in 002a580.

File: docs/src/flags.md (hunk 1 above)
Comment (kdheepak) [reply docs/src/flags.md]: Resolved in 002a580.

**Pull request merged**